### ISSUE TITLE
Document method argument annotations

### ISSUE THREAD

**Issue by blowmage** [googleapis/gapic-generator-ruby#79]
Client method documentation should show whether the arguments are input only, required, or optional. We should not show arguments for fields that are output only.

**Comment by blowmage** [comment]
We have reconsidered the method signatures, and are currently using positional arguments for both the request object/hash and the options object/hash. This means that we are no longer generating named arguments for the request fields. But, users can still make calls using named argument syntax, and Ruby will collect the named arguments into a hash and pass that as the request argument.

We are documenting the fields/named arguments for the request that users can use, and we are excluding the fields with output-only annotations.

**Comment by dazuma** [comment]
@blowmage So is this issue relevant anymore?

**Comment by blowmage** [comment]
I don’t think we document whether an argument’s field is required or not. I’ll look st this tomorrow.

**Comment by blowmage** [comment]
The current generated client method argument documentation does not specify whether the argument (proto message field) is required or optional. It does not, however, document fields that are marked as output-only.

The generated class attribute documentation (proto message) also does not indicate which attributes (proto message fields) are required vs. optional, input vs. output, etc.
